### PR TITLE
BULL repository integration

### PR DESCRIPTION
**Pull request by fborriello**
### :pencil: Description

Adds the bull repository to the one available

### CODE REVIEW

File: src/scripts/build-repo-data.js [hunk 1]
```diff
@@ -28,7 +28,8 @@ const repoNames = [
     { owner: "ExpediaGroup", name: "pitchfork" },
```
Comment (worldtiki) [inline src/scripts/build-repo-data.js]: i suggest we replace this one with pitchfork.
thoughts from the oss folks?

Comment (nikos912000) [reply src/scripts/build-repo-data.js]: Sure.

Comment (fborriello) [reply src/scripts/build-repo-data.js]: @worldtiki I did the requested change

**Review by worldtiki — DISMISSED** [review]

File: src/scripts/build-repo-data.js (hunk 1 above)
Comment (ffeltrinelli) [reply src/scripts/build-repo-data.js]: Previewed locally, looks good :)

Screen Shot 2021-07-14 at 11 46 24 AM

**Review by ffeltrinelli — APPROVED** [review]

File: src/scripts/build-repo-data.js (hunk 1 above)
Comment (ffeltrinelli) [reply src/scripts/build-repo-data.js]: @fborriello would it make sense, for the future, to have a logo with "Expedia Group" instead of "Hotels.com"?

Comment (fborriello) [reply src/scripts/build-repo-data.js]: sure! we even need to change the colour, we should contact the guy who did the image the first time

Comment (fborriello) [reply src/scripts/build-repo-data.js]: @ffeltrinelli I noticed that the build workflow has failed...

Comment (ffeltrinelli) [reply src/scripts/build-repo-data.js]: Yes unfortunately, it's due to recent changes to the org-level permissions, I'll have a look asap! :-)

Comment (ffeltrinelli) [reply src/scripts/build-repo-data.js]: Ok fixed, you are now live! :-)

Comment (fborriello) [reply src/scripts/build-repo-data.js]: Thank you

**Pull request merged**